test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -81,7 +81,17 @@ var helpers = function helpers() {
             for (name in o) {
                 this.writeln(name, o.hasOwnProperty(name) ? "" : " (inherited)", ": ", o[name]);
             }
-        }
+        },
+
+        withPropertyDeleted: function withPropertyDeleted(object, propertyName, callback) {

```

Comment:
If your function name inference is working properly then the `name` will still work. However, it's probably more robust to keep it explicitly for now.